src/core/public/chrome/constants.ts

```diff
@@ -31,3 +31,7 @@
 export const OPENSEARCH_DASHBOARDS_ASK_OPENSEARCH_LINK = 'https://forum.opensearch.org/';
 export const GITHUB_CREATE_ISSUE_LINK =
   'https://github.com/opensearch-project/OpenSearch-Dashboards/issues/new/choose';
+
+export const WORKSPACE_CREATE_APP_ID = 'workspace_create';
+export const WORKSPACE_LIST_APP_ID = 'workspace_list';
+export const WORKSPACE_OVERVIEW_APP_ID = 'workspace_overview';
```

Comment:
There are duplicate constants in Workspace service, we should move these constants to a common directory.

Reply:
@yuye-aws and I discussed this and agreed on this implementation, but after thinking it more throughly, I think this is not ideal. Core app should not aware what's workspace create/list/overview page. 
@SuZhou-Joe Do you think if we can leave it like this for now, and I will make another PR to refactor it.

Reply:
Sure, see the proposal that you want to move the render logic into plugin/workspace.